### PR TITLE
Disable and enable sensitivity recommendations on columns in a database

### PR DESCRIPTION
**Pull request by bashahee**
## Description

Customers will be able to disable (dismiss) or enable sensitivity recommendations on columns in their database. New tests were added in order to verify these changes.

Below issue was opened in order to get a design review:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/379

## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by bashahee** [comment]
@msJinLei can you please merge this PR, as the review was accepted?

**Comment by msJinLei** [comment]
> @msJinLei can you please merge this PR, as the review was accepted?

Still require some time to review the PR.

**Comment by bashahee** [comment]
@msJinLei @markcowl Can this PR be merged?

File: src/Sql/Sql.Test/ScenarioTests/DataClassificationTests.ps1 [hunk 1]
```diff
@@ -354,6 +354,120 @@ function Test-DataClassificationOnSqlDatabase
...
+		Assert-AreEqual $secondSchemaName $recommendation.SchemaName
+		Assert-AreEqual $secondTableName $recommendation.TableName
+		Assert-AreEqual $secondColumnName $recommendation.ColumnName
+		Assert-NotNullOrEmpty $recommendation.InformationType
+		Assert-NotNullOrEmpty $recommendation.SensitivityLabel
+	}
+	finally
+	{
+		# Cleanup
+		Remove-DataClassificationTestEnvironment $testSuffix
```
Comment (msJinLei) [inline src/Sql/Sql.Test/ScenarioTests/DataClassificationTests.ps1]: Don't find the test cases for Enable/Disable-AzSqlInstanceDatabaseSensitivityRecommendation

File: src/Sql/Sql/DataClassification/Cmdlet/ModifyAzSqlDatabaseSensitivityClassificationCmdlet.cs [hunk 2]
```diff
@@ -11,116 +11,19 @@
...
-using Microsoft.Azure.Commands.Sql.Common;
-using Microsoft.Azure.Commands.Sql.Database.Model;
 using Microsoft.Azure.Commands.Sql.DataClassification.Model;
-using Microsoft.Azure.Commands.Sql.DataClassification.Services;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.Sql.DataClassification.Cmdlet
 {
-    public abstract class ModifyAzSqlDatabaseSensitivityClassificationCmdlet : AzureSqlDatabaseCmdletBase<SqlDatabaseSensitivityClassificationModel, DataClassificationAdapter>
+    public abstract class ModifyAzSqlDatabaseSensitivityClassificationCmdlet : ModifyAzSqlDatabaseSensitivityCmdlet
```
Comment (msJinLei) [inline src/Sql/Sql/DataClassification/Cmdlet/ModifyAzSqlDatabaseSensitivityClassificationCmdlet.cs]: Why to  add a new class layer for input object only?

File: src/Sql/Sql/DataClassification/Cmdlet/ModifyAzSqlDatabaseSensitivityRecommendationCmdlet.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+{
+    public abstract class ModifyAzSqlDatabaseSensitivityRecommendationCmdlet : ModifyAzSqlDatabaseSensitivityCmdlet
+    {
+        [Parameter(
+            ParameterSetName = DataClassificationCommon.InputObjectParameterSet,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = DataClassificationCommon.SqlDatabaseSensitivityClassificationObjectHelpMessage)]
+        [ValidateNotNull]
+        public SqlDatabaseSensitivityClassificationModel InputObject { get; set; }
```
Comment (msJinLei) [inline src/Sql/Sql/DataClassification/Cmdlet/ModifyAzSqlDatabaseSensitivityRecommendationCmdlet.cs]: An alias for InputObject as ClassificationObject?

File: src/Sql/Sql/DataClassification/Cmdlet/ModifyAzSqlInstanceDatabaseSensitivityRecommendationCmdlet.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+{
+    public abstract class ModifyAzSqlInstanceDatabaseSensitivityRecommendationCmdlet : ModifyAzSqlInstanceDatabaseSensitivityCmdlet
+    {
+        [Parameter(
+            ParameterSetName = DataClassificationCommon.InputObjectParameterSet,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = DataClassificationCommon.ManagedDatabaseSensitivityClassificationObjectHelpMessage)]
+        [ValidateNotNull]
+        public ManagedDatabaseSensitivityClassificationModel InputObject { get; set; }
```
Comment (msJinLei) [inline src/Sql/Sql/DataClassification/Cmdlet/ModifyAzSqlInstanceDatabaseSensitivityRecommendationCmdlet.cs]: An alias for InputObject?

**Comment by bashahee** [comment]
@msJinLei I will be on vacation during next three weeks, and will address your comments after I come back.
Thanks!

**Comment by markcowl** [comment]
@bashahee Closing this PR as you do not intent to update it in the next 3 weeks.  Please open a new PR, or ask to reopen this one when you are ready.

**Pull request closed without merging**